driver-core/src/main/java/com/datastax/driver/core/exceptions/InvalidConfigurationInQueryException.java

```diff
@@ -22,11 +25,39 @@
  * This is generally throw by query that manipulate the schema (CREATE and
  * ALTER) when the required configuration options are invalid.
  */
-public class InvalidConfigurationInQueryException extends InvalidQueryException {
+public class InvalidConfigurationInQueryException extends InvalidQueryException implements CoordinatorException {
```

Comment:
One small bug here is that `InvalidConfigurationInQueryException` doesn't override the `copy` method from its parent.  So when copy is called on it, you get an instance of `InvalidQueryException` back instead of `InvalidConfigurationInQueryException` when a CONFIG_ERROR is hit.  I think we should fix this but was wondering what you thought.